grpc/generated/protocol/v1/constructors.pb.go

```diff
@@ -0,0 +1,106 @@
+//                           _       _
```

Comment:
This file is not generated by proto?

Reply:
No, there are some additional helper functions that are used in the acceptance tests. However, upon second viewing I will remove this file and simple refactor the test to not use them